geocity/apps/accounts/forms.py

```diff
@@ -171,7 +172,7 @@ def __init__(self, *args, **kwargs):
         required=True,
         label=mark_safe(
             _(
-                'J\'ai lu et j\'accepte la <b><a href="https://geocity.ch/legal/confidentialite.pdf" target="_blank">politique de confidentialité</a></b>'
+                f'J\'ai lu et j\'accepte la <b><a href="{config.PRIVACY_POLICY_URL}" target="_blank">politique de confidentialité</a></b>'
```

Comment:
What happens if not defined?

https://github.com/yverdon/geocity/blob/develop/geocity/apps/core/templates/base_generic.html#L220-L223

And are you sure you want to put the link in bold?

Reply:
It opens the same page again. In bold it's easier to understand this is a link, and shows how important it is